charts/pega/values.yaml

```diff
@@ -421,7 +421,7 @@ installer:
     # For postgres databases that you are upgrading from Pega Infinity version 8.4.0 and later
     # And for Oracle databases that you are upgrading from Pega Infinity version 8.4.3 and later.
     targetDataSchema: ""
-    # Specify the username and password to access the pre-upgraded Pega platform via REST service to perform pre- and post-update actions during zero-downtime updates.
+    # Specify the username and password to access the pre-upgraded Pega Platform via REST service to perform pre- and post-update actions during zero-downtime updates.
```

Comment:
For postgres databases that you are updating from Pega Infinity version 8.4.0 and later.
And for Oracle databases that you are updating from Pega Infinity version 8.4.3 and later.

Specify the username and password that the REST service uses to access the pre-update Pega Platform deployment. The process requires this access to complete pre- and post-update actions during zero-downtime updates.

Reply:
I agree with you that it is inconsistent.

On this entire page, there are:
- 99 references to some form of the word "upgrade"
- 17 references to some form of the word "update"

"upgrade" is is used inconjunction with zero-downtime and also referenced in the title.

I'm happy to make the changes you suggested, but the result will still be inconsistent.